src/Components/Blazor/Build/test/BindRazorIntegrationTest.cs

```diff
@@ -271,6 +271,7 @@ public void Render_BuiltIn_BindToInputWithoutType_WritesAttributes()
         {
             // Arrange
             var component = CompileToComponent(@"
+@using Microsoft.AspNetCore.Components.Web
```

Comment:
Do we need to add this namespace to https://github.com/aspnet/AspNetCore/blob/release/3.0/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/_Imports.razor#L5?

Reply:
Also client side templates?

Reply:
That's wierd, I thought I got them both. Will take a second look 😆

Reply:
Am I crazy? https://github.com/aspnet/AspNetCore/pull/13138/files#diff-ab56cd4be78548275aa24134d6d8b06aR6

Reply:
Guess not. Weird I thought that missing when I first saw it. This looks ok

Reply:
I added it to the classlib template where it was missing.